applicationset/generators/pull_request.go

```diff
@@ -82,10 +82,11 @@ func (g *PullRequestGenerator) GenerateParams(appSetGenerator *argoprojiov1alpha
 
 	for _, pull := range pulls {
 		params = append(params, map[string]string{
-			"number":      strconv.Itoa(pull.Number),
-			"branch":      pull.Branch,
-			"branch_slug": slug.Make(pull.Branch),
-			"head_sha":    pull.HeadSHA,
+			"number":         strconv.Itoa(pull.Number),
+			"branch":         pull.Branch,
+			"branch_slug":    slug.Make(pull.Branch),
+			"head_sha":       pull.HeadSHA,
+			"head_short_sha": pull.HeadSHA[:8],
```

Comment:
Should there be a length check, just in case the API returns an empty (or short) string?